### PR TITLE
886645-Added the Legend UG section for MAUI Charts

### PR DESCRIPTION
**Pull request by NanthiniMahalingam**
Description:
Added the Legend for cartesian, circular, funnel, pyramid and polar chart

### CODE REVIEW

File: MAUI/Cartesian-Charts/Legend.md [hunk 1]
```diff
@@ -0,0 +1,301 @@
...
+    XBindingPath = "XValue",
+    YBindingPath = "YValue",
+    };
+    chart.Series.Add(series);
+
+{% endhighlight %}
+
+{% endtabs %}
+
+## LegendItems Visibility
```
Comment (Saravanan-Madhesh) [inline MAUI/Cartesian-Charts/Legend.md:74]: Dont mention property name in header.

Comment (NanthiniMahalingam) [reply MAUI/Cartesian-Charts/Legend.md]: ok sir. changed sir

File: MAUI/Cartesian-Charts/Legend.md [hunk 2]
```diff
@@ -0,0 +1,301 @@
...
+title: Legend in .NET MAUI Chart control | Syncfusion
+description: This section explains about how to initialize legend and its customization in Syncfusion .NET MAUI Chart (SfCartesianChart) control.
+platform: maui
+control: SfCartesianChart
+documentation: ug
+---
+
+# Legend in .NET MAUI Chart
+
+Legend is used to provide more information about the data point displayed in the chart.
```
[the remark this answers is not in the document]
Comment (NanthiniMahalingam) [reply MAUI/Cartesian-Charts/Legend.md]: Added sir

**Comment by Saravanan-Madhesh** [comment]
@NanthiniMahalingam  give pr to development.

**Comment by NanthiniMahalingam** [comment]
> @NanthiniMahalingam give pr to development.

ok sir

**Pull request closed without merging**